src/DerConverter/Asn/DerAsnUtf8String.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace DerConverter.Asn
+{
+    public class DerAsnUtf8String : DerAsnType
```

Comment:
Please add a Parse and GetBytes unit-test for this type. See https://github.com/huysentruitw/pem-utils/blob/master/tests/DerConverter.Tests/Asn/DerAsnBitStringTests.cs .